backend/kernelCI_app/utils.py

```diff
@@ -69,3 +71,12 @@ def getQueryTimeInterval(**kwargs):
 
 def getErrorResponseBody(reason: str):
     return json.dumps({"error": True, "reason": reason})
+
+
+def string_to_json(string: str) -> dict:
```

Comment:
I would return an `Optional[dict]` here, so we don't return an string by mistake